README.md

```diff
@@ -35,6 +35,8 @@ This API can be used to convert British National Grid eastings and northings to
 To complete the challenge you will need to:
 
 - Write a script called `enrich_data.py` to pull data from the API and enrich the data in the database with the latitudes and longitudes of the captures.
+    - it may take several minutes to receive the data from the API
+    - what 'enrich' means is up to you - you might insert the values into new columns in the table, or perhaps create a new table with the values, or something else
```

Comment:
Can we instead just add `Where you put the data in the database is up to you.` to line 37? Keen to hear how people talk through this and not bias their thinking

Reply:
Sure, it occurred to me that people would prob just grab onto the first suggestion.